PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,4 +6,4 @@ Please make sure your PR meets the following requirements:
 
 The build axis can be controlled by prefixing a ! on the following as appropriate.
 
-MAIN TOMCAT AS_TESTS RTS JACOCO XTS QA_JTA QA_JTS_JACORB QA_JTS_JDKORB QA_JTS_OPENJDKORB BLACKTIE PERF LRA NO_WIN DB_TESTS mysql db2 postgres oracle
+MAIN TOMCAT AS_TESTS RTS JACOCO XTS QA_JTA QA_JTS_JACORB QA_JTS_JDKORB QA_JTS_OPENJDKORB BLACKTIE PERF LRA !NO_WIN DB_TESTS mysql db2 postgres oracle
```

Comment:
Is it good to work with double negation of no win?

Reply:
Good question. I would not object to it being changed to WIN and !WIN but I am not sure if we necessarily need to do that. Thanks for looking!

Reply:
I see, fair enough. I don't count the current state for a big issue.